CMSIS-OS/ChibiOS/ST_NUCLEO_F411RE_EX/README.md

```diff
@@ -0,0 +1,26 @@
+## Configuration of Chibios, HAL and MCU ##
+
+For a successful build the following changes are required:
+
+In _halconf.g_ (in both nanoBooter and nanoCLR folders), when compared with a default file:
+- HAL_USE_SERIAL to TRUE
+- SERIAL_DEFAULT_BITRATE to 115200
+- SERIAL_BUFFERS_SIZE has to be at least 64 (2x the Wire Protocol packet size) otherwise the transmission will be garbled as the packets overrun each other
+
+In _mcuconf.h_ (in both nanoBooter and nanoCLR folders), when compared with a default file:
+- STM32_SERIAL_USE_USART2 to TRUE
+
+In _chconf.h_ (_**only**_ for nanoCLR folder), when compared with a default file:
+- set the CORTEX_VTOR_INIT with the appropriate address of the vector table for nanoCLR
+
+
+## Definitions for each image ## 
+
+Any defines specific to an image (nanoBooter or nanoCLR) that are to be made available throughout the code should be added to the _target_board.h.in_. There is one in [nanoBooter](nanoBooter/target_board.h.in) and another in [nanoCLR](nanoCLR/target_board.h.in) directories, allowing different defines for each one.
+Defines that are common to a target board should be added to the [_target_common.h.in_](target_common.h.in).
+The content of these files is parsed by CMake. It can include CMake variables. Check the documentation of CMake config files [here](https://cmake.org/cmake/help/v3.7/command/configure_file.html?highlight=configure_file).
+
+
+
+
+**NOTE: this configuration was successfully tested in a NUCLEO_F091RC board using the virtual COM port through the ST Link USB connection.**
```

Comment:
I'm probably being picky... but... could you please correct the board name and the description on the connection? I'm under the impression that you are not using the ST Link virtual COM port... 😄